### PR TITLE
XFAIL failing projects

### PR DESCRIPTION
**Pull request by hamishknight**
rdar://109999674
rdar://109999583

### CODE REVIEW

**Comment by hamishknight** [comment]
@swift-ci please test

**Comment by hamishknight** [comment]
Erik is working on a fix for this, so hopefully we won't need this, but going to keep open for now just in case

File: projects.json [hunk 1]
```diff
@@ -973,7 +981,15 @@
         "action": "BuildSwiftPackage",
         "build_tests": "true",
         "configuration": "release",
-        "tags": "sourcekit-disabled swiftpm"
+        "tags": "sourcekit-disabled swiftpm",
+        "xfail": [
+            {
+                "issue": "rdar://109999583",
```
Comment (justice-adams-apple) [inline projects.json:987]: This only affects release builds I believe you'll need
```json
"configuration": "release"
```

File: projects.json [hunk 2]
```diff
@@ -945,7 +945,15 @@
...
```
Comment (justice-adams-apple) [inline projects.json:951]: This only affects debug builds I believe you'll need
```json
"configuration": "debug"
```

**Comment by hamishknight** [comment]
The issue should be fixed by https://github.com/apple/swift/pull/66238, so we shouldn't need this after all

File: projects.json (hunk 1 above)
Comment (hamishknight) [reply projects.json:987]: Good catch! Fortunately I don't think we'll need this PR after all

Comment (justice-adams-apple) [reply projects.json:987]: Even better 👍

**Pull request closed without merging**